relay-event-normalization/src/event.rs

```diff
@@ -154,6 +154,9 @@ pub struct NormalizationConfig<'a> {
     ///
     /// It is persisted into the event payload for correlation.
     pub replay_id: Option<Uuid>,
+
+    /// Controls list of hosts to be excluded from scrubbing
+    pub http_scrubbing_allow_list: Option<Vec<String>>,
```

Comment:
1. This should work with a reference, just like `CombinedMeasurementConfig` above, which contains a reference to global config as well.
2. An empty allowlist and `None` behave the same way, so I think we can skip the `Option`.

```suggestion
    pub span_allowed_hosts: &'a [String]>,
```